lib/serverUtils.js

```diff
@@ -206,24 +208,125 @@ const deleteAccount = function (req, bcrypt, postgres, callback) {
                 });
             });
         }
-        else{
+        else {
             callback(null, -1);
         }
     });
 }
 
+const getCartFromDatabase = function(userId, postgres, callback){
+    // -1 - empty cart else [0:-2] - items [-1] - price
+
+    postgres.query(`SELECT * FROM orders WHERE userid=${userId} and orderstatus=0`, function (err, resultOrder) {
+        if (err){
+            throw err
+        }
+        if (!resultOrder.rows[0]){
+            callback(null, -1)
+        }
+        else{
+            let price = parseInt(resultOrder.rows[0].valueoforder)
+            let orderId = parseInt(resultOrder.rows[0].id) 
+            postgres.query(`SELECT * FROM ordereditems WHERE orderid=${orderId}`), function (err, result){
+                if (err){
+                    throw err
+                }
+                callback(null, result.concate(price))
```

Comment:
Po pierwsze chyba powinno być concat, a po drugie chyba można zrobić to inaczej, po prostu zwrócić coś takiego
`{orderedItems: result, valueOfOrder: price}` i wtedy w callbacku masz ładny obiekt, do którego możesz odwoływać się po kropce.